content/docs/02-guides/06-natural-language-postgres.mdx

```diff
@@ -0,0 +1,683 @@
+---
+title: Natural Language Postgres
+description: Learn how to build a Next.js app that uses AI to interact with a PostgreSQL database using natural language.
+---
+
+# Natural Language Postgres Guide
```

Comment:
What would be the best name for SEO? What would users type? Can we include DB to make it broader?

Reply:
Talk to DB?